backend/capellacollab/feedback/routes.py

```diff
@@ -23,19 +20,6 @@
 router = fastapi.APIRouter()
 
 
-@router.get(
-    "/configurations/feedback",
-    response_model=config_models.FeedbackConfiguration,
-)
-def get_feedback_configuration(
-    db: orm.Session = fastapi.Depends(database.get_db),
-):
-    feedback = config_core.get_global_configuration(db).feedback
-    if not (config.smtp and config.smtp.enabled):
-        util.disable_feedback(feedback)
-    return feedback
-
-
 @router.post(
     "/feedback",
```

Comment:
```suggestion
    "",
```

The feedback key can now be part of the parent `include_router` as prefix.